plugins/logevents/loggrepx_

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+
```

Comment:
How about `set -e` or even `set -eu` in this plugin? (since you used it in the other one)